packages/core/src/__tests__/feature-service-registry.test.ts

```diff
@@ -352,25 +352,74 @@ describe('FeatureServiceRegistry', () => {
       });
     });
 
-    describe('for a Feature Service consumer and optional dependencies', () => {
-      it('creates a bindings object with Feature Services', () => {
-        registry = new FeatureServiceRegistry();
+    describe('for a Feature Service consumer and two optional dependencies', () => {
+      describe('with the first dependency missing', () => {
+        it('creates a bindings object with Feature Services', () => {
+          registry = new FeatureServiceRegistry();
+
+          registry.registerFeatureServices([providerDefinitionA], 'test');
+
+          expect(binderA.mock.calls).toEqual([]);
+
+          expect(
+            registry.bindFeatureServices({
+              id: 'foo',
+              optionalDependencies: {b: '1.0', a: '1.1'}
+            })
+          ).toEqual({
+            featureServices: {a: featureServiceA},
+            unbind: expect.any(Function)
+          });
+
+          expect(binderA.mock.calls).toEqual([['foo']]);
```

Comment:
test b

Reply:
B is not even registered. It is impossible that `binderB` does not exist in this test context.